pkgs/tools/X11/sunpaper/default.nix

```
@@ -0,0 +1,55 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, sunwait
+, wallutils
+, rPackages
+, pkgs
```

```suggestion
, 
```
Please list the package individually.